### PR TITLE
Updating NOTICE to give credit to the CrosVM BSD-3-Clause License.

### PR DESCRIPTION
**Pull request by raduiliescu**
## Reason for This PR

We need to give credit in our release artefacts to the CrosVM BSD-3-Clause license we are carrying it over.

## Description of Changes

Update NOTICE file to include credit for BSD-3-Clause license.

- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

- [x] All commits in this PR are signed (`git commit -s`).
- [x] The reason for this PR is clearly provided (issue no. or explanation).
- [x] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] Any newly added `unsafe` code is properly documented.
- [ ] Any API changes are reflected in `firecracker/swagger.yaml`.
- [ ] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.

### CODE REVIEW

**Review by georgepisaltu — APPROVED** [review]

File: NOTICE [hunk 1]
```diff
@@ -1,2 +1,7 @@
 Firecracker
-Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Copyright 2017-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (raduweiss) [inline NOTICE:2]: Just update it to 2021 directly?

Comment (raduweiss) [inline NOTICE:2]: We also seem to have a lot of files that state 2018/2019 copyright dates, but have been updated in 2020. Should we update those too?

**Review by raduweiss — COMMENTED** [review]
Radu W's comments.

File: NOTICE (hunk 1 above)
Comment (raduiliescu) [reply NOTICE:2]: That is a different task and we should open an issue on that.

Comment (raduiliescu) [reply NOTICE:2]: I don't think is correct to do that. Why not update to 2022?

**Pull request merged**